android/trustwalletcore/build.gradle

```diff
@@ -22,7 +22,7 @@ android {
     }
     externalNativeBuild {
         cmake {
-            version "3.10.2"
```

Comment:
@eddywm could you please confirm? I remember we use a old cmake version here for some reasons

Reply:
ok, I'd revert it back.